allensdk/internal/api/behavior_ophys_api.py

```diff
@@ -49,6 +49,34 @@ def get_stimulus_timestamps(self):
                             .corrected_stim_timestamps)
         return timestamps
 
+    @staticmethod
+    def _process_ophys_plane_timestamps(
+            ophys_timestamps: np.ndarray, plane_group: int,
+            group_count: int):
+        """
+        On mesoscope rigs each frame corresponds to a different imaging plane;
+        the laser moves between N pairs of planes. So, every Nth 2P
+        frame time in the sync file corresponds to a given plane (and
+        its multiplexed pair). The order in which the planes are
+        acquired dictates which timestamps should be assigned to which
+        plane pairs. The planes are acquired in ascending order, where
+        plane_group=0 is the first group of planes.
+
+        Parameters
+        ----------
+        ophys_timestamps: np.ndarray
+            Array of timestamps for 2p data
+        plane_group: int
+            The plane group this experiment belongs to. Signals the
+            order of acquisition.
+        group_count: int
+            The total number of plane groups acquired.
+        """
+        if (group_count == 0) or (plane_group is None):
```

Comment:
~Just a conceptual question - is it right to just return the input here? To me these conditions sound more like the function was supplied junk data, as opposed to the frames don't need to be resampled, so I'd think we should raise an error. But I could be wrong lol~

Oh I just got to the docstrings later in the PR haha it makes sense now!

Reply:
Good point on the order though, may make more sense if I move stuff around a bit.

Reply:
You could even just add a comment to the end of that line that just says something like "to understand this conditional, see the docstrings of those other things." That way people who are lazy and just skipping around looking for answers are explicitly directed there haha

Reply:
Oh darn, they're in separate files. I'll add more info to the docstring.

Reply:
I added some more info to clarify when this doesn't need to be done :)

Reply:
Awesome! :)